packages/core/helper-plugin/src/components/NoContent.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
 
-import { EmptyStateLayout } from '@strapi/design-system';
+import { EmptyStateLayout, EmptyStateLayoutProps } from '@strapi/design-system';
 import { EmptyDocuments } from '@strapi/icons';
-import PropTypes from 'prop-types';
-import { useIntl } from 'react-intl';
+import { MessageDescriptor, PrimitiveType, useIntl } from 'react-intl';
 
-const NoContent = ({ content, ...rest }) => {
+export type NoContentProps = Omit<EmptyStateLayoutProps, 'content' | 'icon'> & {
```

Comment:
Is there any need to export this prop?

Reply:
@madhurisandbhor Indeed there is no reason why to export this prop type. I took a view at the TypeScript migration of the strapi design system and there are all prop types/interfaces exported. I think we should stay at this desicion ✌️

Reply:
Just stepping in, we should always export the types for public APIs (in this case, the props of the component) otherwise you can't reference it elsewhere :)